client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -49,11 +50,37 @@
 
     private final int maxOutstandingCheckpointRequest;
 
+    private final StreamDataRetention retentionConfig;
+
+    /**
+     * The values of this enum indicate if a ReaderGroup chooses Consumption based data retention for Streams it reads from.
+     * For Consumption based retention to work, in addition to setting the appropriate value here,
```

Comment:
"For Consumption based retention to work..."
As this is specified on the ReaderGroup and the group may have multiple streams with different settings this should be more explicit. Spell out what happens if it is not enabled on the stream.

Reply:
fixed.